app/src/main/java/org/oppia/android/app/fragment/InjectableBottomSheetDialogFragment.kt

```diff
@@ -9,7 +9,7 @@ import org.oppia.android.app.view.ViewComponentFactory
 
 /**
  * A fragment that facilitates field injection to children. This fragment can only be used with
- * [org.oppia.android.app.activity.InjectableAppCompatActivity] contexts.
+ * [org.oppia.android.app.activity.InjectableAutoLocalizedAppCompatActivity] contexts.
```

Comment:
Thanks @BenHenning does this means i should update this to InjectableAppCompatActivity? Thanks. My thinking is this can be used with InjectableAutoLocalizedAppCompatActivity or InjectableSystemLocalizedAppCompatActivity now.

Reply:
Updated